src/config/dynamic-theme-fixes.config

```diff
@@ -2910,6 +2910,13 @@ INVERT
 
 ================================
 
+guitarcenter.pl
+
+INVERT
```

Comment:
Change it to
```CSS
IGNORE IMAGE ANALYSIS
table.header td.headerbuttons
```

Reply:
That's not what I figured. Just click at any product from https://guitarcenter.pl/catalog/ - for example https://guitarcenter.pl/catalog/gitary/gitary/gitary-e-akustyczne/sire-larry-carlton-a3-g-vs